content/developer/howtos/discover_js_framework/01_components.rst

```diff
@@ -69,7 +68,8 @@ intended to quickly understand and practice the basics of Owl.
 =======================
 
 As a first exercise, let us implement a counter in the `Playground` component located in
-:file:`owl_playground/static/src/`.
+:file:`owl_playground/static/src/`. To see the result, you can go to the `owl_playground/playground`
+route with your browser.
```

Comment:
No need to apply this suggestion, but it might help you write more concise instructions in your future docs. Avoid "you can go", "you will see", etc.
```suggestion
:file:`owl_playground/static/src/`. Visit the route `/owl_playground/playground` in your browser to see the result.
```